app/views/devise/invitations/edit.html.erb

```diff
@@ -31,7 +31,7 @@
     <div class="registration__field">
       <%= f.form_row :password_confirmation do %>
         <%= f.errors_for :password_confirmation %>
-        <%= f.label :password_confirmation, class: "form__label-heading" %>
+        <%= f.label :password_confirmation, t('authentication.accept_invitation_page.field_labels.password_confirmation'), class: "form__label-heading" %>
```

Comment:
Thanks also @asmega for the heads up on the i18n-debug gem. Good to have in the toolbelt :)